roscala/src/main/scala/coop/rchain/rosette/prim/tuple.scala

```diff
@@ -0,0 +1,28 @@
+package coop.rchain.rosette.prim
+
+import coop.rchain.rosette.{Ctxt, Ob, RblBool, Tuple}
```

Comment:
Still some unnecessary imports. IntelliJ can help you with that: `Code -> Optimize Imports`.

Reply:
we might consider using WartRemover to have it done on compile time (stop compilation if unused imports in source code)

Reply:
Huge wartremover fan here.

Reply:
Done